artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.ScrumIntegration/Features/Project/GetProjectTimeLineResponse.cs

```diff
@@ -1,5 +1,24 @@
 namespace Artificial.Scrum.Master.ScrumIntegration.Features.Project;
 
-internal class GetProjectTimeLineResponse
-{
-}
+public readonly record struct GetProjectTimeLineResponse(
+    int Id,
+    string? EventType,
+    DateTime Created,
+    int ProjectId,
+    string? Comment,
+    string? CommentHtml,
+    // Data
+    int TaskId,
+    string? TaskSubject,
+    int UserStoryId,
+    string? UserStorySubject,
+    // User
+    int UserId,
+    string UserName,
+    string? UserPhoto,
+    string? UserUsername,
+    // Project
+    string ProjectName,
+    // ValuesDiff
+    ValuesDiff? ValuesDiff
```

Comment:
Bardzo mi się nie podoba ta przelotka. To jest 1 do 1 api taigi które jak sam wcześniej zauważyłeś jest słabe. To pole jak np. zmieni się kto jest przypisany do taska to ma ustawione pole jakieś tam, a jak coś innego się zmieni to ma kompletnie inne pola. Przerzucasz odpowiedzialność za poprawne działanie tego na apke frontendową, a backend zostaje dummy przelotką kiedy byłby w stanie ogarnąć to po swojej stronie (np. List<KeyValuePair<string, string>> Differences) i wtedy front dostałby ładnie listę tego co się stało i mógłby to wylistować, a tak to musiałby przejechać po wszystkich kluczach w obiekcie i sprawdzić czy jest nullem (chyba że byś dał typ any zamiast silnie typowanego typescripta ale ja tego na pewno nie przepuszcze na CR). Prośba o przemyślenie tego rozwiązania